spec/Data_Parallelism.tex

```diff
@@ -264,8 +264,8 @@ \section{Forall Intents}
 \end{rationale}
 
 \begin{craychapel}
-An initial implementation of "reduce" intents is also available,
-which permits users to reduce values across iterations of a forall loop.
+To reduce values across iterations of a forall or coforall loop,
+"reduce" intents can be used.
```

Comment:
This looks like it was intended to be a separate commit - was it accidentally committed?  (It seems reasonable to me, I just wanted to make sure you were aware it was included)